Formula/miranda.rb

```diff
@@ -0,0 +1,63 @@
+class Miranda < Formula
+  desc "Lazy, purely functional programming language"
+  homepage "http://miranda.org.uk/"
+  url "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-2066-src.tgz"
+  version "2.066"
+  sha256 "521c281e8c2fde87a2cd7c41d9677daa09514debb71435efc08ff3a7c20016eb"
+
+  depends_on "byacc" => :build
+
+  def install
+    # Make tasks are required to run sequentially
+    ENV.deparallelize
+
+    # Makefile assumes that provided paths exist
+    bin.mkpath
+    lib.mkpath
+    man1.mkpath
+
+    miralib = "#{lib}/miralib"
+
+    args = %W[
+      CC=#{ENV.cc}
+      CFLAGS=-O
+      BIN=#{bin}
+      LIB=#{lib}
+      MAN=#{man1}
+    ]
+
+    inreplace "Makefile" do |s|
+      # Skip compilation of `stdenv.m` & provided examples
```

Comment:
Why?

Reply:
Because it will end up bottled and ultimately cause failures on the first run due to https://github.com/Homebrew/homebrew-core/pull/51003#issuecomment-593660181